packages/nuxt-locokit/src/helpers/transformText.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Allow to create a slug in snake_case from a string
+ */
+export const createSlug = (value: string | null) => {
```

Comment:
Mmm.
Do you think we could add this method in the sdk ? Or in another place, as it's already used in the design system ?
To avoid code duplication.

Reply:
I have this issue with all helper file. I don't know how to fix this with our current organization. I thought about to put in definition package but I am not sure if we have same purpose with this package.

Reply:
Ok, we need to discuss this :-)
I add this to our next DM .

Reply:
Referenced in https://github.com/locokit/locokit/issues/215